Atomic/syscontainers.py

```diff
@@ -963,29 +982,29 @@ def _do_checkout(self, repo, name, img, mode, deployment, values, destination, u
         else:
             systemd_template = SYSTEMD_UNIT_FILE_DEFAULT_TEMPLATE
 
-        values = self._amend_values(values, manifest, name, img, image_id, destination, prefix, unit_file_support_pidfile=SystemContainers._template_support_pidfile(systemd_template))
+        values = self._amend_values(options["values"], manifest, options["name"], options["img"], image_id, options["destination"], options["prefix"], unit_file_support_pidfile=SystemContainers._template_support_pidfile(systemd_template))
```

Comment:
Ouch, just noticed in when I was doing other rebasing, it is interesting that none of our tests actually caught it. Sorry it was my fault for not noticing it.  Will do the fix with options["values"] in pull request: https://github.com/projectatomic/atomic/pull/1160

Reply:
I think we should add some test to verify the expected values for "values" or something like that in the future(either via integration or unit test)  will update this in the issue https://github.com/projectatomic/atomic/issues/1149.

And will assign this to myself once "major refactoring" (I think) is done